File: Java/Isithalloween.java
import java.util.Scanner;

class Isithalloween {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String month = sc.next();
        int day = sc.nextInt();
        System.out.println(month.equals("OCT") && day == 31 || month.equals("DEC") && day == 25 ? "yup" : "nope");

        sc.close();
    }
}

File: C/Isithalloween.c
#include <stdio.h>
#include <string.h>

int main() {
    char month [3];
    int day;
    scanf("%s %d", &month, &day);

    if (strstr(month, "OCT") && day == 31 || strstr(month, "DEC") && day == 25)
        printf("%s", "yup");
    else
        printf("%s", "nope");
}